src/main/grammars/RustParser.bnf

```diff
@@ -308,7 +309,7 @@ ExternAbi ::= extern STRING_LITERAL?
 
 RetType ::= '->' Type { pin = 1 }
 
-fake Function ::=
+Function ::=
   OuterAttr* default_? Vis? FnAttrs FnHeader FnParameters FnTail (';' | InnerAttrsAndBlock)
```

Comment:
`FnAttrs`, `FnHeader` and `FnTail` are used only once, so we can inline them and split the definition of `Function` over several lines.

They don't add clarity, and were used just to remove duplication between various function flavors.